main.tf

```diff
@@ -95,9 +97,12 @@ module "google_mysql_db" {
   read_replica_name_suffix         = local.read_replica_name_suffix
   read_replicas = [
     for array_index in range(var.read_replica_count) : {
-      name = array_index
-      tier = var.instance_size_read_replica
-      zone = local.zone_read_replica
+      name                  = array_index
+      tier                  = var.instance_size_read_replica
+      zone                  = local.zone_read_replica
+      availability_type     = null
+      disk_autoresize       = true
+      disk_autoresize_limit = 0
```

Comment:
These values should come from variables (like it was done for `var.instance_size_read_replica `)

Reply:
disk_autoresize_limit is set to 0 in master (unlimited) so we should keep it this way in read replica also